charts/argo-services/values.yaml

```diff
@@ -1,3 +1,6 @@
 govukEnvironment:
 argocdUrl:
 workflowsNamespace: apps
+rbacTeams:
+  read_only: gov-uk
```

Comment:
I wonder if we should have a list in anticipation of the 3rd team `gov-uk-deployers`, maybe premature.

Reply:
On 2nd pass, I suggest a modification for the second comment of list of auth teams per categories of users. The reason for that is in integration, gov-uk and gov-uk-production will be read-write. I know gov-uk is supposed to be a super-set of gov-uk-production but it might not happen in practice?

Reply:
It would be possible to make it take a list of groups, but it would add complexity. It may be easier to just make sure the team memberships are correct in github